docs/source/reference-core.rst

```diff
@@ -269,7 +269,7 @@ finished`` message.
    the ``with`` statement. This is different from what you might have
    seen with other Python libraries, where timeouts often refer to
    something `more complicated
-   <http://docs.python-requests.org/en/master/user/quickstart/#timeouts>`__. We
+   <https://requests.kennethreitz.org//en/master/user/quickstart/#timeouts>`__. We
```

Comment:
It looks like you have an extra slash here, can you please remove it? Thanks!

Reply:
Fixed now. It was a copy-paste error.